plugins/extractors/postgres/postgres_test.go

```diff
@@ -145,3 +143,76 @@ func execute(db *sql.DB, queries []string) (err error) {
 	}
 	return
 }
+
+func getExpected() []models.Record {
+	return []models.Record{
+		models.NewRecord(&assetsv1beta1.Table{
+			Resource: &commonv1beta1.Resource{
+				Urn:     "postgres::localhost:5438/test_db/article",
+				Name:    "article",
+				Service: "postgres",
+				Type:    "table",
+			},
+			Schema: &facetsv1beta1.Columns{
+				Columns: []*facetsv1beta1.Column{
+					{
+						Name:       "id",
+						DataType:   "bigint",
+						IsNullable: false,
+						Length:     0,
+					},
+					{
+						Name:       "name",
+						DataType:   "character varying",
+						IsNullable: false,
+						Length:     20,
+					},
+				},
+			},
+			Properties: &facetsv1beta1.Properties{
+				Attributes: ut.TryParseMapToProto(map[string]interface{}{
+					"grants": []interface{}{
+						map[string]interface{}{
+							"user":            "test_user",
+							"privilege_types": "INSERT,SELECT,UPDATE,DELETE,TRUNCATE,REFERENCES,TRIGGER",
+						},
+					},
+				}),
+			},
+		}),
+		models.NewRecord(&assetsv1beta1.Table{
+			Resource: &commonv1beta1.Resource{
+				Urn:     "postgres::localhost:5438/test_db/post",
+				Name:    "post",
+				Service: "postgres",
+				Type:    "table",
+			},
+			Schema: &facetsv1beta1.Columns{
+				Columns: []*facetsv1beta1.Column{
+					{
+						Name:       "id",
+						DataType:   "bigint",
+						IsNullable: false,
+						Length:     0,
+					},
+					{
+						Name:       "title",
+						DataType:   "character varying",
+						IsNullable: false,
+						Length:     20,
+					},
+				},
+			},
+			Properties: &facetsv1beta1.Properties{
+				Attributes: ut.TryParseMapToProto(map[string]interface{}{
+					"grants": []interface{}{
+						map[string]interface{}{
+							"user":            "test_user",
+							"privilege_types": "INSERT,SELECT,UPDATE,DELETE,TRUNCATE,REFERENCES,TRIGGER",
```

Comment:
@StewartJingga I tried to use `[]string{"SELECT", "INSERT"}` here, but haven't been able to since the `utils.TryParseMapToProto()` function throws error.

If []string not supported by proto yet??

Reply:
@GrayFlash  btw you should do it like this instead
```
"grants": []map[string]interface{}{
  {
    "user": "test_user",
    ...,
  },
}
```

Reply:
Not sure but when I use it this way, I was getting this error:

`panic: proto: invalid type: []map[string]interface {}`

Reply:
Also I went through this comment on protobuf, I'm checking if there has been any further addition to data types.

https://github.com/golang/protobuf/issues/1302#issuecomment-805453221